lib/LightPanel/LightPanel.js

```diff
@@ -141,26 +141,16 @@ module.exports = kind(
 	deactivated: function () {},
 
 	/**
-	* Determines whether or not we should perform post-transition work.
-	*
-	* @return {Boolean} - If `true`, we should not run the
-	*	{@link enyo.LightPanels#postTransition} method.
-	* @public
-	*/
-	shouldSkipPostTransition: function () {
-		return this._skipPostTransition;
-	},
-
-	/**
 	* This overridable method is called after a transition.
 	*
 	* @public
 	*/
 	postTransition: function () {
-		this.createComponents(this.clientComponents);
-		this.$.client.render();
-		this.$.client.addClass('populated');
-		this._skipPostTransition = true;
+		if (!this.$.client.children.length) {
```

Comment:
Per our discussion on HipChat, we'll merge this for now and take another look later, in the broader context of conditionally creating / diffing components depending on circumstances.